docs/source/quilt_getting_started.ipynb

```diff
@@ -4,18 +4,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Getting Up and Running with Quilt\n",
+    "# Getting Up and Running with Quil-T\n",
     "\n",
     "## Language Documentation\n",
     "\n",
-    "See https://github.com/rigetti/quil/tree/master/rfcs/analog (in particular, `proposal.md` and `spec_changes.md`) for documentation on the Quilt language.\n",
+    "See https://github.com/rigetti/quil/tree/master/rfcs/analog (in particular, `proposal.md` and `spec_changes.md`) for documentation on the Quil-T language.\n",
     "\n",
     "Note that we currently have only a subset of the proposed features in place. In particular,\n",
     "\n",
     "* `RAW-CAPTURE` is not currently supported\n",
     "* classical control operations are not supported\n",
     "\n",
-    "## Install quilt version of PyQuil\n",
+    "## Install quil-t version of PyQuil\n",
     "\n",
     "These are both special branches on the public github repos. Assuming that you are working from your QMI, this could look something like this:\n",
```

Comment:
I suppose this line could use a tweak too?

Reply:
yep!

Reply:
I removed this section entirely since it is unnecessary now that stuff is in master